website/docs/d/iam_trusted_profiles_claim_rule.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_claim_rule"
+description: |-
+  Get information about iam_trusted_profile_claim_rule
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_claim_rule
+
+Provides a read-only data source for iam_trusted_profile_claim_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile claim rules, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule
```

Comment:
Provides a read-only data source for iam_trusted_profile_claim_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile claim rules, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule

as

Retrieve information about IAM trusted profile claim rule as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information, about trusted profile claim rules, see [Create claim rule for a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule)